### PR TITLE
Update de_de.lang

### PR DESCRIPTION
**Pull request by deREALshocK**
Added some German Trans.

### CODE REVIEW

File: de_de.lang [hunk 1]
```diff
@@ -249,4 +249,9 @@ gui.xaero_yaw=Richtung
 gui.xaero_yellow=Gelb
 gui.xaero_zoom=Zoom
 gui.xaero_zoom_in=Minikarte hereinzoomen
-gui.xaero_zoom_out=Minikarte herauszoomen
```
Comment (deREALshocK) [inline de_de.lang]: Mistake Sorry

Comment (thexaero) [reply de_de.lang]: Not a mistake, the difference is the "new line" symbol at the end, so it's all good!

**Comment by thexaero** [comment]
Thank you for the contribution! 
Seems like everything is correct. Are you planning to continue working on this or should I merge?

**Comment by deREALshocK** [comment]
> Thank you for the contribution!
> Seems like everything is correct. Are you planning to continue working on this or should I merge?

Yes i want to continue working on this today thx

**Comment by deREALshocK** [comment]
Done you can merge, i'll translate more things in the future when i have some time. 

again thanks for your awesome mod

File: de_de.lang [hunk 2]
```diff
@@ -244,3 +244,61 @@ gui.xaero_delete_world_msg2=Dies kann nicht rückgängig gemacht werden. Bitte p
...
+gui.xaero_main_entity_as=Spielersymbol als
+gui.xaero_arrow=Pfeil
+gui.xaero_dot=Punkt
+gui.xaero_crosshair=Fadenkreuz
+gui.xaero_cave_maps_depth=Höhlenmodus Tiefe
+gui.xaero_cross_tp=Dimensionsübergreif. TP
+gui.xaero_player_arrow_opacity=Pfeil Deckkraft
+gui.xaero_zoomed_out_enlarged=Herauszoomen beim Vergrößern
+gui.xaero_teleport_default_command=Standard Teleport Befehl
+gui.xaero_waypoints_bottom=Neue WegPunkte oben
```
Comment (thexaero) [inline de_de.lang:271]: "oben" translates as "above" for me. Is this a mistake?

File: de_de.lang [hunk 3]
```diff
@@ -244,3 +244,61 @@ gui.xaero_delete_world_msg2=Dies kann nicht rückgängig gemacht werden. Bitte p
...
+gui.xaero_dot_name_scale=Name Punkt Größe
```
Comment (thexaero) [inline de_de.lang:272]: Not sure if this is correct for ```Dot Name Scale```. Please let me know!

File: de_de.lang [hunk 4]
```diff
@@ -244,3 +244,61 @@ gui.xaero_delete_world_msg2=Dies kann nicht rückgängig gemacht werden. Bitte p
...
+gui.xaero_entity_heads_scale=Objekt Symbol Größe
+gui.xaero_dots_style=Objekt Punkte Größe
```
Comment (thexaero) [inline de_de.lang:274]: I don't think this is correct for ```Entity Dots Style```

File: de_de.lang [hunk 5]
```diff
@@ -244,3 +244,61 @@ gui.xaero_delete_world_msg2=Dies kann nicht rückgängig gemacht werden. Bitte p
...
+gui.xaero_dots_style_default=Standard
+gui.xaero_dots_style_legacy=Vererbt
+gui.xaero_always_entity_nametags=Objektnametags jederz. anzeig.
```
Comment (thexaero) [inline de_de.lang:277]: Please translate what this means for me. I'm not sure if it's correct.

File: de_de.lang [hunk 6]
```diff
@@ -244,3 +244,61 @@ gui.xaero_delete_world_msg2=Dies kann nicht rückgängig gemacht werden. Bitte p
...
+gui.xaero_render_all_wp_sets=Alle WP Sets anzeigen
+gui.xaero_light_overlay_status=Nutze Licht Overlay
```
Comment (thexaero) [inline de_de.lang:279]: Does this mean ```currently using```? Just want to make sure.

**Review by thexaero — COMMENTED** [review]
Thank you very much! Left some comments, but otherwise the translations seem great. Good job!

File: de_de.lang (hunk 2 above)
Comment (deREALshocK) [reply de_de.lang:271]: oh yes thanks for watching sorry its reversed. "Neue Wegpunkte unten" is right. should i change this in the file?

File: de_de.lang (hunk 3 above)
Comment (deREALshocK) [reply de_de.lang:272]: Yes its correct, there are many ways to translate this but the best choice is this cause it would be to long

Comment (thexaero) [reply de_de.lang:272]: Alrighty then!

File: de_de.lang (hunk 4 above)
Comment (deREALshocK) [reply de_de.lang:274]: wow yes you are right thanks idk why i did this mistakes. correct translation is "gui.xaero_dots_style=Objekt Punkte Stil"

File: de_de.lang (hunk 5 above)
Comment (deREALshocK) [reply de_de.lang:277]: I have shortened it because of the Button Sizes etc.
"Objektnametags jederzeit anzeigen"

File: de_de.lang (hunk 6 above)
Comment (deREALshocK) [reply de_de.lang:279]: yes

**Comment by thexaero** [comment]
Awesome! Thank you once again. Merging!

**Pull request merged**